src/plugins/data/public/search/collectors/types.ts

```diff
@@ -6,12 +6,84 @@
  * Side Public License, v 1.
  */
 
+/**
+ * @internal
+ */
 export enum SEARCH_EVENT_TYPE {
+  /**
+   * A search reached the timeout configured in UI setting search:timeout
+   */
   QUERY_TIMED_OUT = 'queryTimedOut',
-  QUERIES_CANCELLED = 'queriesCancelled',
+  /**
+   * The session indicator was automatically brought up because of a long running query
+   */
+  SESSION_INDICATOR_TOUR_LOADING = 'sessionIndicatorTourLoading',
+  /**
+   * The session indicator was automatically brought up because of a restored session
+   */
+  SESSION_INDICATOR_TOUR_RESTORED = 'sessionIndicatorTourRestored',
+  /**
+   * The session indicator was disabled because of a completion timeout
+   */
+  SESSION_INDICATOR_TOUR_DISABLED = 'sessionIndicatorTourDisabled',
```

Comment:
Renamed this to `SESSION_INDICATOR_SAVE_DISABLED` in https://github.com/lukasolson/kibana/pull/18